consensus/reactor.go

```diff
@@ -524,6 +522,7 @@ OUTER_LOOP:
 				msg := &ProposalMessage{Proposal: rs.Proposal}
 				logger.Debug("Sending proposal", "height", prs.Height, "round", prs.Round)
 				if peer.Send(DataChannel, cdc.MustMarshalBinaryBare(msg)) {
+					// NOTE[ZM]: A peer might have received different proposal msg so this Proposal msg will be rejected!
```

Comment:
Is this of any consequence to us? Or just a note that our PRS could be out of date?

Reply:
Just a note. With new mechanism in place we have a mechanism to bring PRS to the right state no matter what Proposal message we get.